package.json

```diff
@@ -42,15 +42,21 @@
     ],
```

Comment:
Odd...  Why was this needed?

Reply:
less|scss|sass i added just to be safe really

`      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg)$": "jest-transform-stub" : I added because I was getting an error where the imported image in some files failed to import properly because of syntax (something along those lines).`"`
      
    The error will look something like this `/src/images/img__ins-and-sm.png:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){PNG....
    
    
    Originally I got it in only my new test, so I added ` "transform": {
        "^.+\\.(js|jsx|ts|tsx)$": "babel-jest"
    },`
    
    To fix my test, and then that cause the error to move towards CentOS, where I then added. this second line.

Reply:
Aha. Ok. I think that the only change to fix that error would maybe just be changing line 45 to:

```
"\\.(css|less|scss|sass|svg|png)$": "identity-obj-proxy",
```

Note the added `svg` and `png`. `jest-transform-stub` does probably something similar, but we don't need to add something new.

Reply:
Ahh that worked, sending that up now, nice catch!

Reply:
Mind also trying to revert the package bumps? We can update those packages, but if it's possible we might want to do that in a separate PR

Reply:
... and the babel config changes. :D

Reply:
just sent up, I'll make another PR for package bumps